src/ScottPlot.Demo/ScottPlot.Demo.csproj

```diff
@@ -11,4 +11,10 @@
     <ProjectReference Include="..\ScottPlot\ScottPlot.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="Images\niceBackground.bmp">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
```

Comment:
Rather than distribute a binary file along with this project, perhaps we could generate a nice background Bitmap procedurally. This could also be used by tests in the future, and wouldn't have to get copied from project to project 🤔 

This comes to mind when I think of test textures 🙃
![image](https://user-images.githubusercontent.com/4165489/85185839-c67feb80-b263-11ea-9874-fffe2f583738.png)

Reply:
Ah, the hammer editor, the bane of my sanity.

I don't know that distributing images with the demo app is too bad, but I could see that it might be useful outside of the demo app, perhaps as part of `ScottPlot.Datagen`. And distributing images (actually anything that isn't .NET Code, even native DLLs) as part of an assembly is often a pain.